src/stores/device/device.ts

```
@@ -0,0 +1,48 @@
+import { makeAutoObservable } from 'mobx';
+import i18n from '~/i18n/react/config';
+import { DeviceMeta, NameTranslations } from './types';
+
+export default class Device {
+  public id: string;
+  public cellIds: string[] = [];
+  public explicit: boolean = false;
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)